homeassistant/components/sensor/influxdb.py

```diff
@@ -100,11 +101,23 @@ def __init__(self, hass, influx_conf, query):
         database = query.get(CONF_DB_NAME)
         self._state = None
         self._hass = hass
+
+        where_clause_template = query.get(CONF_WHERE)
+        _LOGGER.info("Rendering where: %s", where_clause_template)
+        where_clause_template.hass = hass
+        try:
+            where_clause = where_clause_template.render()
```

Comment:
We should render the template each time the sensor updates and we send a new query to influxdb, so add it in `InfluxSensorData`.